commcare_connect/opportunity/views.py

```diff
@@ -135,9 +135,7 @@ def get_context_data(self, **kwargs):
 
         program_invitation_table = None
         if self.request.org_membership and self.request.org_membership.is_admin or self.request.user.is_superuser:
-            program_invitations = ProgramApplication.objects.filter(
-                organization=self.request.org, status=ProgramApplicationStatus.INVITED
-            )
+            program_invitations = ProgramApplication.objects.filter(organization=self.request.org)
```

Comment:
This will show the invitations even when they are accepted. The table was meant to show only the invitation.

Reply:
This change makes all the invites visible for Network Managers. This mitigates the problem of not knowing what Programs a network manager has applied to as the table filters only for Invited.